src/modules/claim/services/claim-verification.service.ts

```diff
@@ -0,0 +1,123 @@
+import { Injectable } from '@nestjs/common';
+import { Logger } from '../../logger/logger.service';
+import { IssuerVerificationService } from './issuer-verification.service';
+import { DIDService } from '../../did/did.service';
+import { RoleCredentialResolver } from '../resolvers/credential.resolver';
+import { ProofVerifier } from '@ew-did-registry/claims';
+import {
+  CredentialResolver,
+  RoleEIP191JWT,
+  isEIP191Jwt,
+} from '@energyweb/vc-verification';
+
+@Injectable()
```

Comment:
@jrhender or @JGiter could you please let me know if my approach to this issue was okay. Not sure if creating/injecting another service (which requires issuer verification service creation as well) is okay.